tickit/adapters/interpreters/command/__init__.py

```diff
@@ -1,4 +1,7 @@
-from tickit.adapters.interpreters.command.command_interpreter import CommandInterpreter
+from tickit.adapters.interpreters.command.command_interpreter import (
+    BeheadingInterpreter,
+    CommandInterpreter,
+)
 from tickit.adapters.interpreters.command.regex_command import RegexCommand
 
-__all__ = ["CommandInterpreter", "RegexCommand"]
+__all__ = ["CommandInterpreter", "RegexCommand", "BeheadingInterpreter"]
```

Comment:
Would this perhaps be better placed at a higher level in the module structure? I don't think anything about it it specifically linked to the command interpreter module

Reply:
Agreed.